presto-product-tests/README.md

```diff
@@ -1,125 +1,106 @@
 # Presto product tests
 
-## Test configuration
+## Installing dependencies
 
-Test assume that you have hadoop and Presto cluster running. 
+*Running product test requires at least 4GB free memory to be available.*
 
-To make it work you need either:
- - define environment variables for hadoop ${HADOOP_MASTER} and Presto ${PRESTO_MASTER} with their IP numbers
- - create test-configuration-local.yaml with following (example content):
+#### On GNU/Linux
+* [```docker >= 1.10```](https://docs.docker.com/installation/#installation)
 
-```
-databases:
-  hive:
-    host: 172.16.2.10
-
-  presto:
-    host: 192.168.205.1
-```
-
-## Running tests
+    ```
+    wget -qO- https://get.docker.com/ | sh
+    ```
 
-Product tests are not run by default. To start them use run following command:
+* [```docker-compose >= 1.60```](https://docs.docker.com/compose/install/)
 
-```
-java -jar target/presto-product-tests-*-executable.jar --config-local tempto-configuration-local.yaml
-```
+    ```
+    pip install docker-compose
```

Comment:
You might want to mention that not all Python installations have pip by default and the user will have to install pip at which point you link to pip's website. For example my Mac Python installation (2.7.10) did not have pip by default and I had to install it using easy_install.